cluster/gce/config-default.sh

```diff
@@ -26,9 +27,11 @@ ZONE=${KUBE_GCE_ZONE:-us-central1-b}
 export REGION=${ZONE%-*}
 RELEASE_REGION_FALLBACK=${RELEASE_REGION_FALLBACK:-false}
 REGIONAL_KUBE_ADDONS=${REGIONAL_KUBE_ADDONS:-true}
+# TODO: Migrate to e2-standard machine family.
 NODE_SIZE=${NODE_SIZE:-n1-standard-2}
 NUM_NODES=${NUM_NODES:-3}
 NUM_WINDOWS_NODES=${NUM_WINDOWS_NODES:-0}
+# TODO: Migrate to e2-standard machine family.
 MASTER_SIZE=${MASTER_SIZE:-n1-standard-$(get-master-size)}
```

Comment:
So that may turn out to be `n1-standard-2` or `-1`

Reply:
Good find. Looks like for 3 node tests we should expect n1-standard-1: https://github.com/kubernetes/cloud-provider-gcp/blob/3ab8427dd3fe122b1901011782794eebdb176b65/cluster/gce/config-common.sh#L29